utils/fhirpath/Package.md

```diff
@@ -0,0 +1,10 @@
+# FHIR R4 Utils FHIRPath package
+
+## Package Overview
+
+Package containing FHIR related processors and utilities for implementing FHIR Path related integrations.
+
+|                      |                      |
+|----------------------|----------------------|
+| FHIR version         | R4                   |
+| Implementation Guide | http://hl7.org/fhir/ |
```

Comment:
Shouldn't this be https://hl7.org/fhir/fhirpath.html?

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerinax-health.fhir.r4/commit/08e57e7fae8edd666b262458106711c51a4fd471